qa/rpc-tests/txoutsbyaddress.sh

```diff
@@ -0,0 +1,163 @@
+#!/usr/bin/env bash
```

Comment:
Yes, please look at the existing tests in `qa/rpc-tests`, use the Python framework. Also add the test to `qa/pull-tester/rpc-tests.py` so it gets executed by default.